PluginsAndFeatures/azure-toolkit-for-intellij/gradle.properties

```diff
@@ -5,14 +5,14 @@ org.gradle.jvmargs='-Duser.language=en'
 sources=false
 
 intellij_version=IC-231.8109-EAP-CANDIDATE-SNAPSHOT
```

Comment:
Where can I find the right version number?

Reply:
I usually pick the latest snapshot from https://www.jetbrains.com/intellij-repository/snapshots that has a version number as close to Rider's as possible.